Library/Homebrew/test/test_pkg_version.rb

```diff
@@ -35,7 +35,8 @@ def test_to_s
     assert_equal "1.0_1", PkgVersion.new(Version.new("1.0"), 1).to_s
     assert_equal "1.0", PkgVersion.new(Version.new("1.0"), 0).to_s
     assert_equal "1.0", PkgVersion.new(Version.new("1.0"), 0).to_s
-    assert_equal "HEAD", PkgVersion.new(Version.new("HEAD"), 1).to_s
+    assert_equal "HEAD_1", PkgVersion.new(Version.new("HEAD"), 1).to_s
+    assert_equal "HEAD-ffffff_1", PkgVersion.new(Version.new("HEAD-ffffff"), 1).to_s
```

Comment:
These new cases look like they should be using `Version.create`, not `Version.new`. Or am I misunderstanding the purpose of this test?